easybuild/easyconfigs/l/libevent/libevent-2.0.22-GCC-5.4.0-2.26.eb

```diff
@@ -0,0 +1,19 @@
+easyblock = 'ConfigureMake'
+
+name = 'libevent'
+version = '2.0.22'
+
+homepage = 'http://libevent.org/'
+description = """The libevent API provides a mechanism to execute a callback function when a specific
+ event occurs on a file descriptor or after a timeout has been reached.
+ Furthermore, libevent also support callbacks due to signals or regular timeouts."""
+
+toolchain = {'name': 'GCC', 'version': '5.4.0-2.26'}
+
+source_urls = [
+    'https://github.com/%(name)s/%(name)s/releases/download/release-%(version)s-stable/',
```

Comment:
@edmondac any particular reason you changed the `source_urls` compared to that of the existing easyconfigs?

```python
source_urls = [
    'https://github.com/downloads/%(name)s/%(name)s/',
    'https://sourceforge.net/projects/levent/files/%(name)s/%(name)s-%(version_major_minor)s/',
]
```